### PR TITLE
Update remove-former-employee.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to correct the order and match it with individual pages

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7675

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/add-users/remove-former-employee.md [hunk 1]
```diff
@@ -82,12 +82,11 @@ After you remove a user, you have up to 30 days to restore their account.
...
-|[Step 4 - Give another employee access to OneDrive and Outlook data](remove-former-employee-step-4.md)|If you only remove a user's license but don't delete the account, the content in the user's OneDrive will remain accessible to you even after 30 days. <p> Before you delete the account, you should give access of their OneDrive and Outlook to another user. After you delete an employee's account, the content in their OneDrive and Outlook is retained for **30** days. During that 30 days, however, you can restore the user's account, and gain access to their content. If you restore the user's account, the OneDrive and Outlook content will remain accessible to you even after 30 days.|
-|[Step 5 - Wipe and block a former employee's mobile device](remove-former-employee-step-5.md)|Removes your business data from the phone or tablet.|
+|[Step 3 - Wipe and block a former employee's mobile device](remove-former-employee-step-5.md)|Removes your business data from the phone or tablet.|
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee.md]:
```suggestion
|[Step 3 - Wipe and block a former employee's mobile device](remove-former-employee-step-3.md)|Removes your business data from the phone or tablet.|
```

File: microsoft-365/admin/add-users/remove-former-employee.md [hunk 2]
```diff
@@ -82,12 +82,11 @@ After you remove a user, you have up to 30 days to restore their account.
...
+|[Step 4 -   - Forward a former employee's email to another employee or convert to a shared mailbox](remove-former-employee-step-3.md)|This lets you keep the former employee's email address active. If you have customers or partners still sending email to the former employee's address, this gets them to the person taking over the work.|
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee.md]:
```suggestion
|[Step 4 - Forward a former employee's email to another employee or convert to a shared mailbox](remove-former-employee-step-4.md)|This lets you keep the former employee's email address active. If you have customers or partners still sending email to the former employee's address, this gets them to the person taking over the work.|
```

File: microsoft-365/admin/add-users/remove-former-employee.md [hunk 3]
```diff
@@ -82,12 +82,11 @@ After you remove a user, you have up to 30 days to restore their account.
...
+|[Step 5 - Give another employee access to OneDrive and Outlook data](remove-former-employee-step-4.md)|If you only remove a user's license but don't delete the account, the content in the user's OneDrive will remain accessible to you even after 30 days. <p> Before you delete the account, you should give access of their OneDrive and Outlook to another user. After you delete an employee's account, the content in their OneDrive and Outlook is retained for **30** days. During that 30 days, however, you can restore the user's account, and gain access to their content. If you restore the user's account, the OneDrive and Outlook content will remain accessible to you even after 30 days.|
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/add-users/remove-former-employee.md]:
```suggestion
|[Step 5 - Give another employee access to OneDrive and Outlook data](remove-former-employee-step-5.md)|If you only remove a user's license but don't delete the account, the content in the user's OneDrive will remain accessible to you even after 30 days. <p> Before you delete the account, you should give access of their OneDrive and Outlook to another user. After you delete an employee's account, the content in their OneDrive and Outlook is retained for **30** days. During that 30 days, however, you can restore the user's account, and gain access to their content. If you restore the user's account, the OneDrive and Outlook content will remain accessible to you even after 30 days.| 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**